src/android/io/cozy/plugins/listlibraryitems/ListLibraryItems.java

```diff
@@ -299,16 +304,29 @@ protected Integer doInBackground(UploadFilePayload... uploadFilePayloads) {
                     huc.setRequestProperty(key, val);
                 }
                 huc.setRequestProperty("Content-Length", "" + FILE_SZ);
```

Comment:
I think you can remove this `content-length` option which is set by the `setFixedLengthStreamingMode`